### ISSUE TITLE
Installing a custom pack

### ISSUE THREAD

**Issue by Venom-git** [steven-dfheinz/dfhz_ddp_mpack#3]
Hey Steve,

I am trying to use Ambari -followed with kafka, ranger . however i am new to this and i wanted to use a custom stack . i am using ubuntu and amazon linux. i have came to a dead end. Please let me know how i can reach you out. You can reach me on this a/c itself

**Comment by steven-matison** [comment]
Yes ambari is a dead end.   This repo was a way to still use ambari,  however, fast forward 2 years,  this is no longer an option.   It is possible to install ambari with last free version of hdp (look around my repos) but I highly recommend that you do not take this path.  Check out apache bigtop or look at CDP if you need turnkey solution.
